### DISCUSSION TITLE
Inserting Null In a binding for a mySQL database

### DISCUSSION THREAD

**Discussion by aeroptl** [Budibase/budibase#13068] [Help] [answered]
Hello, 

I'm trying to change a numerical value to NULL on a mySQL database which is connect to budibase. 

Whenever I try do this with an execute query action on the design section, it returns an error : Incorrect integer value: '' for column 'user_id' at row 1

Is there another way around to achieve this or does Budibase need to fix this in an update? 

Thanks.

**Comment by ConorWebb96** [comment, marked as answer]
Hey @aeroptl,

The fact that the value is returning `""` in your error suggests that you are passing back a string and not a NULL.

Can you show the query you are using? Screenshooting and providing the code? The query i've used below seems to work in my case. Therefore I'm going to assume the issue is likely related to your query and how you are NULLing the field.

```sql
UPDATE number_checker
SET number_field = NULL
WHERE id = 1;
```

<img width="1249" alt="Screenshot 2024-02-20 at 08 19 24" src="https://github.com/Budibase/budibase/assets/126772285/04a6ee0e-d4c1-4116-8d33-8f8babcf31d3">

Hopefully this helps.

**Comment by aeroptl** [comment]
Hi,

Yes perfect i realised what I did wrong, it should have been just passed at
the query rather than making the binding and setting the value to NULL.

Thanks!
